CHANGES.rst

```diff
@@ -22,7 +22,12 @@ New Features
 Breaking Changes
 ^^^^^^^^^^^^^^^^
 
-- TBD
+- Rework the Docker image tagging scheme: ``isso:latest`` is now rebuilt on
+  every push to ``master``, while ``isso:release`` points to the latest stable
+  release (`#970`_, janw). Previously, ``:latest`` pointed to the latest stable
```

Comment:
Was this ever actually true?

Reply:
Yup, that is back from when I manually released docker images to the `ghcr.io/isso-comments` namespace. Not particularly knowing any better, using `:lastest` was the tagging strategy I implemented for releases. I used `:dev` for images off of `master`.

Reply:
Ah, fair enough